article/conf/application.conf

```diff
@@ -16,6 +16,14 @@ akka {
       }
     }
   }
+
+  blocking-operations {
```

Comment:
Here we used the same values as: 

https://github.com/guardian/frontend/blob/main/admin/conf/application.conf#L26
https://github.com/guardian/frontend/blob/main/facia/conf/application.conf#L27

Reply:
What does the code do, I gather it's to do with some kind of concurrency?

Reply:
This creates a dedicated thread pool for handling thread pool operations off the main thread. Here is the documentation: https://doc.akka.io/docs/akka/current/typed/dispatchers.html

Reply:
`facia` also has a thread pool of size 128. We checked the size of the facia instances and it's the same as `article`, so don't expect to see any issues with that.